scripts/train_deepmass.py

```diff
@@ -0,0 +1,182 @@
+# Script for training a DeepMass inference model
+import os
+
+os.environ['XLA_FLAGS']='--xla_gpu_cuda_data_dir=/gpfslocalsys/cuda/10.1.2'
+
+from absl import app
+from absl import flags
+
+import haiku as hk
+import jax
+import jax.numpy as jnp
+from jax.experimental import optix
+
+import numpy as onp
+import pickle
+from functools import partial
+from astropy.io import fits
+
+from flax.metrics import tensorboard
+
+# Import tensorflow for dataset creation and manipulation
+import tensorflow.compat.v2 as tf
+tf.enable_v2_behavior()
+import tensorflow_datasets as tfds
+
+from jax_lensing.models.convdae import SmallUResNet
+from jax_lensing.models.convdae2 import MediumUResNet
```

Comment:
hummmm what's the difference between convdae and convdae2 again? can we remove one of them?